### PR TITLE
Second assault map weapons

### PR DESCRIPTION
**Pull request by peter-budo**
Need help with unlocks requirements/criteria label

• Added missing images for second assault
• Fixed weapon scaling
• Added string resources as much possible

@karllindmark can you have look at unlocks and help me fix missing requirement labels?

### CODE REVIEW

**Comment by karllindmark** [comment]
@peter_budo: Is it the Strings you're looking for?

**Comment by karllindmark** [comment]
@peter-budo: Try it out now. Don't merge though, as I haven't had time to go through the other changes. However, my latest commit should show the new criterias (I hope - worked for GOL Magnum at least).

**Comment by peter-budo** [comment]
@karllindmark ready for merge as unlock weapons now list requirements as necessary. Can't start build on PR since I'm on restrictive wi-fi

File: BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/stats/weapons/WeaponStatsAdapter.java [hunk 1]
```diff
@@ -35,7 +35,10 @@ public View getView(int position, View view, ViewGroup parent) {
         Weapon weapon = itemsList.get(position);
 
         ((TextView) view.findViewById(R.id.index)).setText(String.valueOf(position + 1));
```
Comment (karllindmark) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/stats/weapons/WeaponStatsAdapter.java:37]: Can't you use setText(View, int, String) here instead? It looks cleaner and should be in BaseIntelAdapter. :-) :+1:

Comment (peter-budo) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/stats/weapons/WeaponStatsAdapter.java:37]: I would not say it is cleaner, it is more hiding of what it is doing...

Comment (karllindmark) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/ui/stats/weapons/WeaponStatsAdapter.java:37]: Also outside the scope of this PR. :-)

**Pull request merged**